catalog/publishers.jsonld

```diff
@@ -3,27 +3,27 @@
   "@graph": [
     {
       "@id": "http://www.getty.edu/research/",
-      "identifier": "Getty"
+      "alternateName": "Getty"
     },
     {
       "@id": "https://www.cultureelerfgoed.nl",
-      "identifier": "RCE"
+      "alternateName": "RCE"
     },
     {
       "@id": "http://www.wereldculturen.nl",
-      "identifier": "NMVW"
+      "alternateName": "NMVW"
     },
     {
       "@id": "http://data.bibliotheken.nl/doc/thes/p075301482",
-      "identifier": "KB"
+      "alternateName": "KB"
     },
     {
       "@id": "https://rkd.nl",
-      "identifier": "RKD"
+      "alternateName": "RKD"
     },
     {
       "@id": "https://www.wikidata.org",
-      "identifier": "Wikidata"
+      "alternateName": "Wikidata"
```

Comment:
Not really an `alternameName` but rather the `name`. For consistency I decided to use `alternateName` here anyway.

Reply:
Makes sense!